src/base/_typography.scss

```diff
@@ -22,6 +22,7 @@ html {
     breakpoint.$xs,
     breakpoint.$xl
   ); /* 1 */
+  // stylelint-disable-next-line property-no-unknown
```

Comment:
This is an unknown property, but we've got a comment above explaining why we're using it, so I'm just telling Stylelint to ignore this one.